gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -36,6 +36,11 @@
 #define NNS_TENSOR_SIZE_LIMIT_STR	"16"
 #define NNS_TENSOR_DIM_NULL ({0, 0, 0, 0})
 
+/**
+ * @brief The maximum rank in meta info (see GstTensorMetaInfo).
```

Comment:
Add more description (to avoid confusion with RANK for other/tensor(s) GST CAP):
```
* This RANK is applied to meta info of other/tensors-flexible only and does not affect other/tensors(s)'s NNS_TENSOR_RANK_LIMIT. 
```

Reply:
updated